var/spack/repos/builtin/packages/py-py-spy/package.py

```diff
@@ -17,3 +17,4 @@ class PyPySpy(PythonPackage):
     # Need to figure out how to manage these with Spack once we have a
     # CargoPackage base class.
     depends_on('rust', type='build')
+    depends_on('libunwind')
```

Comment:
```suggestion
    depends_on('unwind')
```
`unwind` is a virtual package provided by `libunwind` (on Linux) and `apple-libunwind` (on macOS). `libunwind` causes a lot of build issues on macOS so we use a system library instead.

Reply:
Suggestion applied manually on the pull request (I didn't see the "Commit suggestion" button)